### PR TITLE
[feature] Extended OPENWISP_USERS_EXPORT_USERS_COMMAND_CONFIG

### PR DESCRIPTION
**Pull request by pandafy**
Added 'registered_user.method' and 'registed_user.is_verified' to OPENWISP_USERS_EXPORT_USERS_COMMAND_CONFIG setting.

**Blockers**
- [ ] https://github.com/openwisp/openwisp-users/pull/356

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (pandafy) [inline .github/workflows/ci.yml:48]: TODO: Remove before merging

**Review by nemesifier — COMMENTED** [review]
This looks good, let's wait until the PR which is depended upon is merged and we can then rollback the ci.yml change and merge this too.

**Review by nemesifier — APPROVED** [review]

**Pull request merged**
